tests/unit/GithubContext/Application/CommandHandler/Inspection/SendViolationsAsCommentsTest.php

```diff
@@ -86,4 +86,21 @@ public function testAnInspectionWithNoReportTriggersNothing()
 
         $this->handler->handle($command);
     }
+
+    public function testAnInspectionForAFlightModeRepositoryTriggersNothing()
+    {
+        $inspection = $this->createMock(PullRequestInspection::class);
+        $repository = $this->createMock(Repository::class);
+
+        $inspection->method('hasReport')->willReturn(true);
+        $inspection->method('getRepository')->willReturn($repository);
+
+        $repository->method('isFlightModeEnabled')->willReturn(true);
+
+        $this->reporter->expects($this->never())->method('report');
+
+        $command = new Command\Inspection\SendViolationsAsComments($inspection);
```

Comment:
Parameter #1 $inspection of class Regis\GithubContext\Application\Command\Inspection\SendViolationsAsComments constructor expects Regis\GithubContext\Domain\Entity\PullRequestInspection, PHPUnit_Framework_MockObject_MockObject given.